webapp/drizzle/0004_chilly_ben_urich.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS "freeze_proposals" (
```

Comment:
This migration and 0005_wakeful_quasimodo.sql

Look like the same one. Maybe this was not removed by mistake?

Reply:
Fixed